src/components/ListItem.jsx

```diff
@@ -1,9 +1,46 @@
 import './ListItem.css';
+import { useEffect } from 'react';
+import { updateItem } from '../api/firebase';
+
+export function ListItem({ listToken, item }) {
+	const millisecondsIn24hrs = 84600000;
+	useEffect(() => {
+		if (item.isChecked) {
+			let calcDate =
+				item.dateLastPurchased.seconds * 1000 + millisecondsIn24hrs;
+
+			setTimeout(() => {
+				updateItem(listToken, item, {
+					isChecked: !item.isChecked,
+				});
+			}, calcDate);
+		}
+	}, [item, listToken]);
+	//
+	const handleChange = () => {
+		if (item.isChecked === false) {
```

Comment:
I see that I am not able to uncheck the item. What if I accidentally clicked it? We need to cover that situation as well :)

Reply:
We were told to leave it uncheckable for a future issue